app/uk/gov/hmrc/exports/controllers/NotificationsController.scala

```diff
@@ -109,7 +109,7 @@ class NotificationsController @Inject()(
 
     parseXmlResult match {
       case Success(metaData) =>
-        val mrn = metaData.declaration.flatMap(_.functionalReferenceId)
+        val mrn = metaData.response.head.declaration.flatMap(_.id)
```

Comment:
Please don't use `head`. Use `headOption` it'll be safer, when this field will be empty